network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/text_only_teaser_block.html

```diff
@@ -2,23 +2,21 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
 {% block block_content %}
-  <div class="tw-row tw-py-4">
-	{% for block in self.cards %}
-      {% with card=block %}
-        <div class="tw-px-4 medium:tw-w-1/3 tw-mb-5">
-          <div class="tw-bg-white tw-relative tw-py-4 tw-w-full tw-flex tw-flex-col">
-            {% if card.heading_link %}<a href="{% pageurl card.heading_link %}">{% endif %}
-            <span class="tw-h5-heading tw-mb-2">{{ card.heading }}</span>
+  <section class="tw-container tw-mb-7">
+    <div class="tw-row tw-grid medium:tw-grid-cols-3 tw-gap-x-4 tw-gap-y-5 medium:tw-gap-6">
+      {% for block in self.cards %}
+        {% with card=block %}
+          <div class="tw-pt-4 tw-relative tw-w-full tw-flex tw-flex-col tw-border-t tw-border-black/10">
+            {% if card.heading_link %}<a class="hover:tw-text-inherit" href="{% pageurl card.heading_link %}">{% endif %}
+            <div class="tw-h5-heading tw-mb-2">{{ card.heading }}</div>
             {% if card.heading_link %}</a>{% endif %}
-              {% if card.meta_data %}
-                <div class="tw-flex tw-flex-wrap">
-                  <span class="tw-h6-heading tw-text-gray-40 tw-py-1 tw-mb-1">{{ card.meta_data }}</span>
-                </div>
-              {% endif %}
-            {{ card.body }}
+            {% if card.meta_data %}
+              <span class="tw-body-small tw-pb-2">{{ card.meta_data }}</span>
```

Comment:
I feel like I am missing something, but should this be a `div` if it's on its own line?

Reply:
In this context either one is fine. I could make it a div so that it's clearer that its own it's own line.

Reply:
This has been done now

Reply:
It's a pattern I have used more often. I was wondering if I was missing something about `span`s that would make them more proper element.